app/templates/sitewide/flat_ui_template.html

```diff
@@ -1,5 +1,4 @@
-{% load static %}
-<!DOCTYPE html>
```

Comment:
I might be mistaken, but in some web browsers if `DOCTYPE html` isn't the very first line in the file it causes issues. Reverting this change just in case (even though it looks ugly)